apps/api/serializers.py

```diff
@@ -1,6 +1,12 @@
+from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
-from .models import Entry, Notification, PrivateMessage, Tag, User
+from entries.models import Tag, Entry
+from messeges.models import PrivateMessage
+from notifications.models import Notification
+
+
+User = get_user_model()
```

Comment:
User is not used in this file, please remove it.

Reply:
ok i'll do that